apps/ideas/forms.py

```diff
@@ -334,22 +317,21 @@ def helper(self):
             crisp.layout.Field,
             template="bootstrap3/user_checkboxselectmultiple_field.html",
         )
-        helper['invites'].wrap(
-            crisp.layout.Field,
-            template="bootstrap3/user_checkboxselectmultiple_field.html",
-        )
         return helper
 
+    def random_avatar(self):
```

Comment:
I think much cooler than a random avatar would be an avatar based on the email address given. Something quick would be computing an adler checksum `zlib.adler32(bytes(email, 'UTF-8')) % 4`.